lib/resources/command.rb

```diff
@@ -28,6 +28,9 @@ def initialize(cmd)
     end
 
     def result
+      if @command.nil?
+        raise 'InSpec `command` was called with `nil` as the argument. This is not supported. Please provide a valid command instead.'
```

Comment:
Actually, let's not skip. Skip points to unsupported use on a platform, at least in the current context. This however is a full user-error and should never be done this way. It's like adding nil to a number ;) We need the exception handling piece but it needs to fail afaics.
I can move it to initialize as well, don't mind either way :)